### DISCUSSION TITLE
Use Flat Colors instead of Gradients

### DISCUSSION THREAD

**Discussion by PlaidDroid** [ezra-bible-app/ezra-bible-app#1077] [Ideas]
First of all, I found this application just today and already this is the best desktop bible app I've used! Thanks for putting this out into the world!

One suggestion I had is that if we could swap out the gradient style in the tabs and buttons, with using just flat colors. Personally I think, that the gradient feels a bit outdated; and also I guess as per more recent design systems. A reference would be [Google's Material design](https://m3.material.io/components). Though this might be a minute scrutiny, I feel it would make a difference, in the sense that the app would feel more premium, in par with commercial software. (branding - that's one area where I've seen opensource sometimes is weak vs. commercial products).

This should be a low priority though, since improving the app functionality-wise should be first.

I'll try to dip my feet more into the codebase, to see if I can help too (not a pro web dev though).

**Comment by tobias-klein** [comment]
@PlaidDroid Thanks for your positive feedback and thanks for the suggestions.
I converted the issue to a discussion since I feel like this is a bit of an experimental topic considering my comments below:

Your suggestion is related to #250. Currently the app uses a very old design based on [JQuery UI](https://jqueryui.com/). That is 10-15 years old!

The reason that this app uses JQuery UI is that I originally created it in 2009/2010 as a web app and then ported it to Electron a few years ago while keeping the old frontend code.

There are two ways to improve:
1. Hack the existing JQuery UI CSS styles.
1. Replace JQuery UI with a more modern CSS framework (see #250).

The JQuery UI themes are here (Cupertino, Dark-Hive):
https://github.com/ezra-bible-app/ezra-bible-app/tree/master/css/jquery-ui

Option 2) is quite a lot of effort. Option 1) could be possible, but may not be trivial either. Generally option 1) would be more sustainable in the long run.

There is a list of JQuery UI themes available here:
https://jqueryui.com/themeroller/

It may be possible to integrate other themes into the app. However, conceptually we only differentiate between 1 light color theme (Cupertino) and 1 dark color theme (Dark-Hive) at the moment.

We are not even using the latest version of JQuery UI, but the older version 1.8.24.
At some point I tried upgrading to a newer version, but I had so much trouble (specifically with the JQuery UI SelectMenu plugin) that I gave up on it. In any case - JQuery UI is not really maintained anymore today.

More suggestions and experiments are welcome!
If you would like to contribute feel free to experiment and open a pull request.
You should start with the BUILD instructions [here](https://github.com/ezra-bible-app/ezra-bible-app/blob/master/BUILD.md), so that you can locally experiment with your own changes.
